### DISCUSSION TITLE
Java as a new language for Language Tab

### DISCUSSION THREAD

**Discussion by gii-pan** [PaloAltoNetworks/docusaurus-openapi-docs#394] [Ideas]
I really appreciate the LanguageTab and CodeTab components to show examples of requests. 
I would like to know if Java could be the 8th language.
I looked into the code and there are [OkHttp, Unirest and Java itself](https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/blob/main/packages/docusaurus-theme-openapi-docs/src/theme/ApiDemoPanel/Curl/languages.json), but in the documentation Java is not mentioned :/

**Comment by sserrata** [comment]
Hi @gii-pan, right now we are selectively including supported languages to help keep the bundle size low. That said, we are using a forked version of [postman-code-generators](https://github.com/postmanlabs/postman-code-generators#:~:text=List%20of%20supported%20code%20generators%3A) so it's definitely possible to support Java.

**Reply by gii-pan** [reply]
Hello @sserrata, thank you for your time! I understand your point.  ☺

Do you think in a next release would accept java? Or for now just the seven languages?

The project I'm working on would be complete with the java example request.

**Comment by sserrata** [comment]
Hi @gii-pan, is there a preference for `OkHttp` or `Unirest`? Currently, the plugin is opinionated about which `variant` to support for each language. I think for now we can go with the `variant` that is most popular for `java`, if that makes sense. Then, we can look toward supporting all variants in our upcoming v2.0.0 release. Let me know what you think:

OkHttp example:

<img width="723" alt="Screenshot 2023-02-06 at 10 44 13 AM" src="https://user-images.githubusercontent.com/9343811/217017053-c31b6804-9c63-42e5-9c19-d46cf6a1eddc.png">

**Reply by gii-pan** [reply]
Hello @sserrata, we don't have a preference, you can choose the one which is more useful and easy to implement! 

Thank you for the OkHttp example! We're awaiting to see Java 😍

Just another question: is it possible to have code snippet with Spring? I don't know how the postman plugin works, but it's worth to ask.

We're looking forward for v2.0.0 🤩

**Comment by gii-pan** [comment]
Hello @sserrata, 

I was looking in our website and the C#, PHP and Java are without any theme.
Just like this:
![image](https://user-images.githubusercontent.com/64155751/221922973-9b73a82e-6f53-4cce-bf52-71a39077b186.png)

Do you know what can possible be?
